### ISSUE TITLE
Create tutorials about Path / PathFollow (3D) 

### ISSUE THREAD

**Issue by rogeriodec** [godotengine/godot-docs#2242]
There is a lack of information on how to use Path / PathFollow in 3D.
Since the 2D version is quite different, most commands do not work in 3D.

**Comment by cbscribe** [comment]
As near as I can tell in my testing Path/PathFollow work exactly the same as Path2D/PathFollow2D. The methods and properties are even identically named.

The one major difference is in actually drawing the path, which is more difficult in 3D because you don't have any gizmos on the points/handles to accurately set their positions. This seems like a usability issue to me, which is addressed in https://github.com/godotengine/godot/issues/27024

In the interim perhaps an example in `Curve3D` on how to programmatically generate a curve would be helpful.

**Comment by rogeriodec** [comment]
Yeah, this issue was created by me. But the GREATEST problem with Godot documentation is actually the lack of EXAMPLES. The learning curve would be exponentially increased if all documented methods had some practical EXAMPLES.
